README.md

```diff
@@ -19,25 +19,70 @@ yarn add --dev @netlify/ipx
 Create `netlify/functions/ipx.ts`:
 
 ```ts
-import { createIPXHandler } from '@netlify/ipx'
+import { createIPXHandler } from "@netlify/ipx";
 
 export const handler = createIPXHandler({
-  domains: ['images.unsplash.com']
-})
+  domains: ["images.unsplash.com"],
+});
 ```
 
 Now you can use IPX to optimize both local and remote assets ✨
 
 Resize `/test.jpg` (in `dist`):
 
 ```html
-<img src="/.netlify/functions/ipx/w_200/static/test.jpg"/>
+<img src="/.netlify/functions/ipx/w_200/static/test.jpg" />
 ```
 
 Resize and change format for a remote url:
 
 ```html
-<img src="/.netlify/functions/ipx/f_webp,w_450/https://images.unsplash.com/photo-1514888286974-6c03e2ca1dba"/>
+<img
+  src="/.netlify/functions/ipx/f_webp,w_450/https://images.unsplash.com/photo-1514888286974-6c03e2ca1dba"
+/>
+```
+
+## Remote Patterns
+
+Instead of setting an allowlist on `domains`, you may wish to use the option `remotePatterns`. This method allows wildcards for hostname as well as pathname segments.
```

Comment:
[sand] Consider linking to the remotePatterns documentation at https://github.com/vercel/next.js/blob/canary/docs/api-reference/next/image.md#remote-patterns

Reply:
The feature in this repo can be used without next. I will be submitting a separate PR to our next plugin to upgrade `@netlify/ipx` to this version and passing along the values. Although the same format is used, next&netlify users do not have to know about this repo.